README.md

```diff
@@ -82,6 +82,15 @@ cookiecutter gh:elixir-cloud-aai/cookiecutter-python
 > `update-template` job from `update.yaml` github action workflow file as
 > `cookiecutter` doesn't support autosync like Cruft.
 
+### Extra dependency
+
+Some checks and configuration might require additional dependencies.
+While we try to keep the template as minimal as possible and have a fallback
+option to use the default configuration, we recommend installing the following:
+
+- Requests: `pip install requests`
+- Beautiful Soup: `pip install beautifulsoup4`
```

Comment:
Whether added here or elsewhere, you probably want to add a comment explaining what these optional dependencies are used for.